src/Juvix/Pipeline.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE LiberalTypeSynonyms #-}
+
+module Juvix.Pipeline where
+
+import qualified Juvix.Core as Core
+import qualified Juvix.Frontend as Frontend
+import qualified Juvix.FrontendContextualise.InfixPrecedence.Environment as Target
+import Juvix.Library
+import Prelude (String)
+
+data Error
+  = PipeLine Core.Error
+  | ParseErr String
+  deriving (Show)
+
+toCore :: [FilePath] -> IO (Either Error Target.FinalContext)
+toCore paths = do
+  x <- Frontend.ofPath paths
+  case x of
+    Left er -> pure (Left (ParseErr er))
+    Right x ->
+      case Core.ofFrontend x of
+        Left errr -> pure (Left (PipeLine errr))
+        Right con -> pure (Right con)
```

Comment:
```suggestion
      pure $ case Core.ofFrontend x of
        Left errr -> Left (PipeLine errr)
        Right con -> Right con
```

Reply:
Will do, but first I'll see how our formatter handles this

Reply:
Works fine, I moved it even higher though